lib/tasks/deposit.rake

```diff
@@ -8,20 +8,10 @@ task complete_deposits: :environment do
   abort 'ERROR: This task only runs in the development environment!' unless Rails.env.development?
```

Comment:
Since there is now a service (and less copypasta) the comment could be updated.

Reply:
Will do, thx!

Reply:
I took a look at this, and I think we probably still want this task to only run in the development environment, since it forces depositing objects into the deposited state and jams in a fake druid. Does that make sense?

Reply:
Yes, that makes sense. I was commenting more about the part that says "We could consider extracting the common bits to a service?" since that is what this PR does. Unless you think there is some other further work to do here. Your call.

Reply:
Ah! GitHub UI confused my eyes. Yes, thank you, that defunct comment is now removed. Good eye. :)